.jshintrc

```diff
@@ -62,6 +64,8 @@
 
     "maxlen": 110,
 
+    "varstmt": true,
```

Comment:
It seems like this is the style we switched to in #62 (except that pull didn’t update the tests 🤷), so I went ahead and formalized it here.

(This does the opposite of what it sounds like — it *disallows* the `var` keyword, so you can only use `let` or `const`.)